### PR TITLE
Make functional Overview GUI

### LINKED ISSUE #91: As a user, I can see an overview of the cases in the current list
so that I can get a sense of the total number of cases at a glance.

### LINKED ISSUE #101: As a user, I can organize the overview by location
so that I can see and compare the number of dengue cases in different regions.

### LINKED ISSUE #102: As a user, I can organize the overview by age group
so that I can see and compare the number of dengue cases for different ages.

### LINKED ISSUE #103: As a user, I can organize the overview by dengue variant
so that I can see and compare the number of dengue cases for each variant.

### PR DESCRIPTION
**Pull request by axmszr**
Base GUI of the Overview.
Supports the three types (age, postal, variant), with postal default upon start-up.
Updated Location to show postal code prefixes.
Updated SampleDataUtil to match new restrictions.

Closes #91, closes #101, closes #102, closes #103

### CODE REVIEW

File: src/main/java/seedu/dengue/model/overview/AgeOverview.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import seedu.dengue.model.person.Person;
+
+/**
+ * An {@code Overview} that runs analysis by {@code Age}.
+ *
+ * @see AgeAnalyst
+ */
+public class AgeOverview extends Overview {
+    private static final String AGE_TITLE = "Overview by Age";
+    private static final int MAX_AGE_LENGTH = 9; // "321 - 123"
```
Comment (sumhungyee) [inline src/main/java/seedu/dengue/model/overview/AgeOverview.java:17]: not sure if comment should remain here

File: src/main/java/seedu/dengue/model/overview/Overview.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import seedu.dengue.logic.analyst.Analyst;
+import seedu.dengue.logic.analyst.DataBin;
+import seedu.dengue.model.person.Person;
+
+/**
+ * The API of the Overview list.
+ */
+public abstract class Overview {
+    protected static final int MAX_INDEX_LEN = 4; // _12.
+    protected static final String GAP = "     "; // whitespace of length 3
```
Comment (sumhungyee) [inline src/main/java/seedu/dengue/model/overview/Overview.java:16]: not sure if comments should remain

**Review by sumhungyee — APPROVED** [review]
LGTM, thanks for correcting static variable format

File: src/main/java/seedu/dengue/model/overview/AgeOverview.java (hunk 1 above)
Comment (axmszr) [reply src/main/java/seedu/dengue/model/overview/AgeOverview.java:17]: Thanks for pointing it out! I'll remove them in the next PR.

**Pull request merged**